specs/commandline/verify.md

```diff
@@ -58,6 +58,8 @@ Use `notation certificate` command to configure trust stores.
 
 Users who consume signed artifact from a registry use the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. The trust policy is a JSON document. User needs to create a file named `trustpolicy.json` under `{NOTATION_CONFIG}`. See [Notation Directory Structure](https://notaryproject.dev/docs/tutorials/directory-structure/) for `{NOTATION_CONFIG}`.
 
+**Note** To circumvent an issue in go-asn1-ber/asn1-ber where it allocates large amount of memory for decoding ber, notation does not support x509 subjects that contain "=#". For more information please look at [this issue](https://github.com/notaryproject/notation-go/issues/276).
```

Comment:
```suggestion
**Note** Notation does not support x509 certificates whose subject contains `=#`. For more information please look at [this](https://github.com/notaryproject/notation-go/issues/276) issue.
```

Reply:
i feel like it's better to call out limitations relevant to commands rather than hoping the customer checks the limitations before using notation, so will add in sign also.